tests/test_helpers.py

```diff
@@ -9,6 +9,9 @@
     (["helper", "a_proprev", "A. Einstein", "nsf", "2020-04-08", "-q",
       "Tess Guebre", "--status", "downloaded", "--title", "A flat world theory"],
      "A. Einstein proposal has been added/updated in proposal reviews\n"),
+    (["helper", "a_manurev", "A. Einstein", "2020-09-15", "--requester",
+      "Niels Bohr", "--reviewer", "zcliu"],
```

Comment:
I don't think you are testing all of the optional arguments, but please do.

Reply:
Partially completed. All optional arguments are tested except "database", which I am not quite sure how to test.

Reply:
you don't have to test that one.